style.css

```diff
@@ -1,16 +1,23 @@
 * { 
```

Comment:
Please don't use "*" selector. If you want to make something on level higher then "body" use "html"
"*" - means everything